views/default/admin/statistics/user.php

```diff
@@ -1,16 +1,58 @@
 <?php
 // Work out number of users
-$users_stats = get_number_users();
-$total_users = get_number_users(true);
+$options = [
+	'type' => 'user',
+	'sybtype' => 'user',
+	'count'=> true,
+];
+
+// Banned user count
+$options['metadata_name_value_pairs'] = ['name' => 'banned', 'value' => 'yes'];
+$users_banned = elgg_get_entities($options);
+
+// Active user count
+$options['metadata_name_value_pairs'] = ['name' => 'banned', 'value' => 'no'];
+$users_active = elgg_get_entities($options);
+
+$hidden_status = access_get_show_hidden_status();
+access_show_hidden_entities(true);
+
+// Disabled user count
+$users_disabled = elgg_get_entities($options) - $users_active;
```

Comment:
This can give you an incorrect number ($users_active is not ALL enabled users)

There should be some magic available so you can use the `metadata_name_value_pairs` to filter `enabled => false` to get the disabled entities

Reply:
`'enabled' => 'no'`... not false